<ui version="4.0" >
 <class>DifferentiateCurvesDialog</class>
 <widget class="QDialog" name="DifferentiateCurvesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Differentiate Curves</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="cycleOrderGroup" >
     <property name="title" >
      <string>Property Selection and Cycle Order</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item row="1" column="1" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item row="1" column="1" >
         <widget class="QPushButton" name="_up" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>Raise in Selection Order: Alt+up</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="shortcut" >
           <string>Alt+Up</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QPushButton" name="_down" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>Lower in Selection Order: Alt+down</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="shortcut" >
           <string>Alt+Down</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QPushButton" name="_add" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>Select: Alt+s</string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QPushButton" name="_remove" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>Remove: Alt+r</string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item rowspan="3" row="0" column="0" >
       <layout class="QVBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Available:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="_availableListBox" />
        </item>
       </layout>
      </item>
      <item rowspan="3" row="0" column="2" >
       <layout class="QVBoxLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel2" >
          <property name="text" >
           <string>Selected:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="_selectedListBox" />
        </item>
       </layout>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>21</width>
          <height>61</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>51</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsGroup" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>16</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item row="0" column="1" >
         <widget class="QSpinBox" name="_maxLineWidth" >
          <property name="minimum" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="textLabel3" >
          <property name="text" >
           <string>Maximum line width:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2" >
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <widget class="QLabel" name="textLabel4" >
          <property name="text" >
           <string>Point density:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QComboBox" name="pointDensity" >
          <item>
           <property name="text" >
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>High</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Medium</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Low</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="3" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_buttonGroupRepeat" >
     <property name="title" >
      <string>Repeat Across</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="_radioButtonRepeatPlot" >
        <property name="text" >
         <string>S&amp;ingle Plot</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_radioButtonRepeatWindow" >
        <property name="text" >
         <string>Single &amp;Window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_radioButtonRepeatApplication" >
        <property name="text" >
         <string>&amp;All Windows</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_buttonGroupApplyTo" >
     <property name="title" >
      <string>Apply To</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="_radioButtonApplyWindow" >
        <property name="text" >
         <string>Current Window</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="_radioButtonApplyAllWindows" >
        <property name="text" >
         <string>All Windows</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item row="0" column="2" >
      <widget class="QPushButton" name="Cancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="OK" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>_radioButtonRepeatPlot</tabstop>
  <tabstop>_radioButtonRepeatWindow</tabstop>
  <tabstop>_radioButtonRepeatApplication</tabstop>
  <tabstop>_radioButtonApplyWindow</tabstop>
  <tabstop>_radioButtonApplyAllWindows</tabstop>
  <tabstop>OK</tabstop>
  <tabstop>Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
